chalicelib_fourfront/checks/wrangler_checks.py

```diff
@@ -1051,14 +1051,53 @@ def find_entrez_gene_status(report):
     return None
 
 
-@check_function()
+@check_function(add_to_ignore= None, rm_from_ignore=None)
 def validate_entrez_geneids(connection, **kwargs):
     ''' query ncbi to see if geneids are valid
     '''
     check = CheckResult(connection, 'validate_entrez_geneids')
     # add random wait to stagger runs with other checks on this schedule
     wait = round(random.uniform(0.1, random_wait), 1)
     time.sleep(wait)
+
+    # get any geneids to ignore
+    last_result = check.get_primary_result()
+    # hack to add empty full_output for first time it is used
+    # last_result['full_output'] = {}
+    # if last one was fail, find an earlier check with non-FAIL status
+    it = 0
+    while last_result['status'] == 'ERROR' or not last_result['kwargs'].get('primary'):
+        it += 1
+        # this is a daily check, so look for checks with 12h iteration
+        hours = it * 12
+        last_result = check.get_closest_result(diff_hours=hours)
```

Comment:
I am getting this far in my test run when I run into an exception (before reaching the 100 hours) in `get_closest_result`--maybe it needs to be caught so it can continue? The exception: `Could not find closest non-ERROR result for prefix: validate_entrez_geneids. Attempted with 12 diff hours and 0 diff mins.`

It also gave me a couple other questions: do we have a successful result within 100 hours? Or how do we resolve that 100 hour error if it occurs?